412.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {number} n
+ * @return {string[]}
+ */
+var fizzBuzz = function(n) {
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```
It would be best to use `const` for arrays unless the array needs to be re-assigned after the initialization.